src/components/UI/organisms/AddMealForm/AddMealForm.jsx

```diff
@@ -0,0 +1,77 @@
+import { Box, Button, FormHelperText, Stack, TextField, Typography } from '@mui/material';
+import { addMealSchema } from '../../../../schemas/addMealSchema';
+import { useForm } from 'react-hook-form';
+import './AddMealForm.css';
+import { yupResolver } from '@hookform/resolvers/yup';
+
+function AddMealForm() {
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm({
+    mode: 'onBlur',
+    resolver: yupResolver(addMealSchema),
+  });
+
+  const onSubmit = (data) => console.log(data);
+  return (
+    <>
+      <Typography className="AddMealForm__header">Add Meal Form</Typography>
+      <Box className="AddMealForm" component="form" onSubmit={handleSubmit(onSubmit)}>
```

Comment:
I think it'd be best if this form used also `noValidate` to prevent default browser validation as there's already the one prepared by you. This default validation is used now if you click the submit button and one of the inputs if empty.